buildifier/buildifier.go

```diff
@@ -136,9 +136,7 @@ func main() {
 
 	diff = differ.Find()
 
-	// TODO(bazel-team): Handle "-" as stdin/stdout mode too.
-
-	if len(args) == 0 {
+	if (len(args) == 0) || (len(args) == 1 && args[0] == "-") {
```

Comment:
yes, the right most paren clarify the precedence of || and && so are needed, but the leftmost are extraneous and should be removed.